42577 Request - Auto Release of Web Orders

tyndmar opened the pull request:
This is the "guts" of changing order status to Hold if one or more tests fails
/testers/smart files that allow manipulation of the validationTest table - this table is an extensible list of tests that must pass to approve orders
custom/oe-ctrl.w - selects tables to display to a user Admin to turn on/off validation testing
system/oeValidate.p - structured procedure containing the specific test procedures
system/tagProcs.p - modifications to support Hold tags

Commit:
42577 Request - Auto Release of Web Orders
This is the "guts" of changing order status to Hold if one or more tests fails
/testers/smart files that allow manipulation of the validationTest table - this table is an extensible list of tests that must pass to approve orders
custom/oe-ctrl.w - selects tables to display to a user Admin to turn on/off validation testing
system/oeValidate.p - structured procedure containing the specific test procedures
system/tagProcs.p - modifications to support Hold tags

brad-advantzware reviewed (changes requested):
Changes requested documented in ticket

Commit:
42577 Request - Auto Release of Web Orders

Commit:
Merge branch 'develop' into feature/42577_Request_-_Auto_Release_of_Web_Orders

Commit:
42577 Request - Auto Release of Web Orders
See Zoho ticket for details

Commit:
Files changed:
browsers/brwTags.w - rearranged display
oe/v-ord.w - reconfigured logic to conform with new procedure structure
sos/sdoComProcs.i - removed diagnostic message
system/oeValidate.p - see spreadsheet
system/TagProcs.p - see spreadsheet
